src/Ombi/ClientApp/src/app/discover/components/card/discover-card.component.html

```diff
@@ -3,7 +3,7 @@
 <div id="result{{result.id}}" *ngIf="fullyLoaded" class="ombi-card dark-card c" [style.display]="hide ? 'none' : 'block'">
     <div class="card-top-info">
         <div class="top-left" id="type{{result.id}}">
-            {{ 'Common.' + RequestType[result.type] | translate }}
+            {{ result.title }}
```

Comment:
I don't think we spoke about this, the text should say what type of media it is.